src/utils/web.py

```diff
@@ -121,8 +137,8 @@ def strError(e):
 # application-specific function.  Feel free to use a callable here.
 proxy = None
 
-def getUrlFd(url, headers=None, data=None, timeout=None):
-    """getUrlFd(url, headers=None, data=None, timeout=None)
+def getUrlFd(url, headers=None, data=None, timeout=None, noverify=None):
+    """getUrlFd(url, headers=None, data=None, timeout=None, noverify=None)
```

Comment:
Please don't use double negatives. Rename the variable to `checkCertificates`.
You should also use a boolean, and make it default to True.